tests/acceptance/test_issue_details.py

```diff
@@ -105,7 +105,7 @@ def test_javascript_specific_event(self):
         self.page.visit_issue(self.org.slug, event.group.id)
         self.browser.snapshot("issue details javascript - event details")
 
-        self.browser.find_element_by_xpath("//button//span[contains(text(), 'curl')]").click()
+        self.browser.click('[data-test-id="curl"]')
```

Comment:
We shouldn't need a `data-test-id` for Button, you can use `aria-label`

```suggestion
        self.browser.click('[aria-label="curl"]')
```

Reply:
I just checked this and it wasn’t adding an aria-label to the button until I added the text translation thingy?

Reply:
So since we don’t want `curl` to be translated, I added the data-test-id instead

Reply:
Tell me if I’m being a pillock here though

Reply:
@robinrendle Hmm, it should add an `aria-label` w/o the translation - I think `<Button>` has the same issue with AlertLink where it does not pass the `data-test-id` prop down.